src/openzaak/components/catalogi/models/zaaktype.py

```diff
@@ -297,6 +298,10 @@ def save(self, *args, **kwargs):
         if not self.pk:
             transaction.on_commit(AutorisatieSpec.sync)
 
+        if self.selectielijst_procestype and not self.selectielijst_procestype_jaar:
+            response = requests.get(self.selectielijst_procestype).json()
```

Comment:
Please use the `Client` abstraction like in https://github.com/open-zaak/open-zaak/blob/main/src/openzaak/components/catalogi/admin/forms.py#L150 to do the actual retrieval - this future-proofs us for proxies like NLX and allows us to configure authorization if it's relevant (for example, because the API gateway requires it).

Raw `requests` usage is pretty much never needed.